core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -270,12 +239,12 @@ default boolean isComplete() {
      *   <li>Other exceptions that occurred due to an error while retrieving the elements.</li>
      * </ul>
      *
-     * @param withPooledObjects if {@code true}, receives the pooled {@link ByteBuf} and {@link ByteBufHolder}
-     *                          as is, without making a copy. If you don't know what this means, use
-     *                          {@link StreamMessage#drainAll()}.
+     * @param options {@link SubscriptionOption}s to subscribe with. Note that
+     *                {@link SubscriptionOption#NOTIFY_CANCELLATION} is ineffective because there's no
+     *                cancelling while draining all elements.
      * @return the {@link CompletableFuture} which will be completed with the list of the elements retrieved.
      */
-    CompletableFuture<List<T>> drainAll(boolean withPooledObjects);
+    CompletableFuture<List<T>> drainAll(SubscriptionOption... options);
```

Comment:
I changed this to take the `SubscriptionOption`s even though I don't have to do because we don't call `Subscription.cancel()` inside.
I thought it would be better for consistency with the `subscribe` methods and we might add more options to `SubscipriptionOption` later. So I changed it, but if you think we don't have to, just let me know. I will undo it. :-)